src/extensions/workspace/workspace.ts

```diff
@@ -16,7 +16,7 @@ export default class Workspace implements ComponentHost {
     /**
      * private access to the legacy consumer instance.
      */
-    private consumer: Consumer,
+    readonly consumer: Consumer,
```

Comment:
I think it's better to keep it private and add a getter called _consumer, so people will know they shouldn't use it.